pkg/scheduler/framework/plugins/nodevolumelimits/csi.go

```diff
@@ -155,7 +155,7 @@ func (pl *CSILimits) filterAttachableVolumes(
 
 		driverName, volumeHandle := pl.getCSIDriverInfo(csiNode, pvc)
 		if driverName == "" || volumeHandle == "" {
-			klog.V(5).Infof("Could not find a CSI driver name or volume handle, not counting volume")
```

Comment:
Actually, these could be changed to structured logging `InfoS` (and below, the `Errorf` could be `ErrorS`) (see https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/1602-structured-logging)

Reply:
Yes, but for those cases, I cannot see any benefit after migrating to structured logging, 

see the non-goals in that spec:
https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/1602-structured-logging#non-goals

> We are not replacing currently used logging library (klog) or the way in which it is used
We are not preserving full compatibility of logging output
We are not proposing structuring all logs in Kubernetes

Reply:
unless `klog` is going to be deprecated and retied, I think we won't bother to migrating those.